interview-questions.md

```diff
@@ -44,12 +75,12 @@ Researched answer:
 
 ## Looking Ahead: Terms for Next Week
 
```

Comment:
Reviewing the terms and looking over the syllabus can be very helpful understanding the concepts from the lectures.